integrations/logs/sentry-cocoa-swiftlog/README.md

```diff
@@ -0,0 +1,187 @@
+# Sentry Swift-Log Integration
+
+A `swift-log` handler that forwards log entries to Sentry's structured logging system, automatically capturing application logs with full context including metadata, source location, and log levels.
+
+## Installation
+
+### Swift Package Manager
+
+Add the following dependencies to your `Package.swift`:
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/getsentry/sentry-cocoa", from: "9.0.0"),
```

Comment:
`m`: Is this one really necessary? Wouldn't it be resolve as a transitive dependency anyways?